docx/bookmark.py

```diff
@@ -4,7 +4,11 @@
 
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-from collections import Sequence
+try:
+    from collections import Sequence
+except DeprecationWarning:
```

Comment:
I think we should reverse this.

```
try:
    from collections.abc import Sequence
except ImportError:
  from collections import Sequence
```
Because in 3.9 importing directly from collections will result in an import error, which is not caught here, also, as far as I can tell, you cannot catch a warning: https://docs.python.org/3/library/warnings.html (it's more like a logging statement).

Reply:
yeah, didn't know that. thanks for explaining.